### PR TITLE
made auton delay

### PR DESCRIPTION
**Pull request by Griffin9881**
### Pull Request

---------------------------------------------

#### Issue Number

Closes #74

### CODE REVIEW

**Review by reediculous456 — CHANGES_REQUESTED** [review]
Did you see the code that I pushed? I think it will be a lot easier to work with from a DS standpoint and be more flexible

**Comment by Griffin9881** [comment]
where did you push that code?

**Comment by reediculous456** [comment]
https://github.com/LakotaRobotics1038/2024RobotProject/tree/auton-delay

**Comment by Griffin9881** [comment]
I feel like a dropdown would be fine since you wouldn't really have a situation where you needed to be more accurate than that

**Comment by reediculous456** [comment]
Interesting, I would expect that to work

**Comment by Griffin9881** [comment]
me too

**Comment by reediculous456** [comment]
Does my branch work? I have a theory

**Comment by Griffin9881** [comment]
both branches work now because i guess it had to be in the auton class, I would prefer to use my code because I think it would be easier than using a text box and also it was made by a student

**Comment by reediculous456** [comment]
Ok. One thing you could do to reduce the overhead here is remove the enum and just assign the seconds straight to the options in the chooser. The only reason we don’t do that for autons is because we need to construct the auton in auto init not on robot init

**Review by matunkar — DISMISSED** [review]

File: src/main/java/frc/robot/autons/AutonSelector.java [hunk 1]
```diff
@@ -21,6 +21,7 @@ public enum AutonChoices {
 
     // Choosers
     SendableChooser<AutonChoices> autoChooser;
+    SendableChooser<String> delayChooser;
```
Comment (reediculous456) [inline src/main/java/frc/robot/autons/AutonSelector.java:24]:
```suggestion
    SendableChooser<double> delayChooser;
```
If you do this you don’t need the switch statement. You can just return the get value from the chooser

**Review by reediculous456 — CHANGES_REQUESTED** [review]

**Review by reediculous456 — APPROVED** [review]

**Pull request merged**